lang-js/nvm/autobuild/build

```diff
@@ -1,13 +1,15 @@
-abinfo "Creating nvm profile..."
+abinfo "Creating nvm bashrc..."
 
 cat << 'EOF' > aosc_nvm_stub.sh
 export NVM_DIR="${HOME}/.nvm"
 
 EOF
 cat aosc_nvm_stub.sh nvm.sh > nvm_aosc.sh
 
-abinfo "Installing nvm profile, bash completion and doc..."
-install -Dm644 nvm_aosc.sh "${PKGDIR}/etc/profile.d/30-nvm.sh"
-install -Dm644 bash_completion "${PKGDIR}/usr/share/bash-completion/completions/nvm"
-install -d "${PKGDIR}/usr/share/doc/nvm/"
-cp *.md "${PKGDIR}/usr/share/doc/nvm/"
+abinfo "Installing nvm shell startup files, bash completion and doc..."
+install -Dvm644 nvm_aosc.sh "${PKGDIR}/etc/bashrc.d/30-nvm.sh"
```

Comment:
Any rationale on why this export should be a bash specifc thing? Looks like NVM_DIR should be set regardless of default shell choice.

Also note that desktop environments generally don't source `/etc/bashrc.d` but would include `/etc/profile.d`. If this environment variable were to be used by any graphical applications (say, an IDE or a graphical text editor) they will no longer see `NVM_DIR` in their environment.